packages/components/src/Banner/Banner.stories.tsx

```diff
@@ -52,6 +52,7 @@ const dialogArgs = {
 
 export const Brand = Template.bind(null);
 Brand.args = {
+  className: "visible md:invisible",
```

Comment:
to show `.visible` and breakpoints working even tho `.visible` exists in both Bootstrap and TW

Reply:
Nice! There's no Bootstrap in this repo though, so it wouldn't be a problem here :') we should test this in `traject` though, where Bootstrap is imported

Reply:
oooop right